src/System.Numerics.Tensors/System.Numerics.Tensors.csproj

```diff
@@ -0,0 +1,48 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\tools\common.props" />
+  <PropertyGroup>
+    <Description>Tensor class which represents and extends multi-dimensional arrays</Description>
+    <TargetFramework>netstandard1.1</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <PackageTags>.NET Tensor Array corefxlab</PackageTags>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="System\Numerics\ArrayUtilities.cs" />
```

Comment:
Yes, we do.  I've disable the wildcard include because I want to apply the code-generator metadata to the TT-generated items.  See: EnableDefaultCompileItems